### PR TITLE
Added support for channel 14 in wifi access point mode.

### LINKED ISSUE #2913: WiFi Channels 12-14 not usable in AP mode
**Describe the bug**
When the max WiFi channel is set to 14, channels 12-14 cannot be used in access point mode.
The table with the possible channels has always 11 rows.

In station mode, it possible to connect to access point using channels 12-14.

**To Reproduce**
1. Go to Network/wlan0/wireless
2. Click on Wireless => Access point
3. Scroll down to "Channels"
4. No channels 12,13,14 available

**Expected behavior**
There should be 3 additional rows for the 12, 13, 14

**Target Environment (please complete the following information):**
 - Board I.MX6
 - OS version: Debian 
 - Additional info: Kura 4.0.0

**Additional context**

### PR DESCRIPTION
**Pull request by MMaiero**
Added support for channel 14 in wifi access point mode.

**Related Issue:** This PR fixes/closes #2913 

**Description of the solution adopted:** Channel 14, if supported, is now selectable and configurable

**Screenshots:** N/A

**Any side note on the changes made:** N/A

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java [hunk 1]
```diff
@@ -117,7 +117,7 @@
 
     private static final String REGEX_PASSWORD_WPA = "^[ -~]{8,63}$";
     private static final String REGEX_PASSWORD_WEP = "^(?:[\\x00-\\x7F]{5}|[\\x00-\\x7F]{13}|[a-fA-F0-9]{10}|[a-fA-F0-9]{26})$";
-    private static final int MAX_WIFI_CHANNEL = 13;
+    private static final int MAX_WIFI_CHANNEL = 14;
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java:120]: I think you forgot to change the header in this file, it should be "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtWifiChannel.java [hunk 2]
```diff
@@ -25,7 +25,8 @@
     netWifiChannel10,
     netWifiChannel11,
     netWifiChannel12,
-    netWifiChannel13;
+    netWifiChannel13,
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtWifiChannel.java:28]: I think you forgot to change the header in this file, it should be "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"

File: kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/HostapdConfigWriter.java [hunk 3]
```diff
@@ -316,11 +316,11 @@ private String updateInterfaceAndDriver(String interfaceName, String interfaceDr
     }
 
     private String updateChannels(WifiConfig wifiConfig, String fileAsString) throws KuraException {
-        if (wifiConfig.getChannels()[0] > 0 && wifiConfig.getChannels()[0] < 14) {
+        if (wifiConfig.getChannels()[0] > 0 && wifiConfig.getChannels()[0] <= 14) {
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/HostapdConfigWriter.java:319]: I think you forgot to change the header in this file, it should be "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"

**Review by LeoNerdoG — COMMENTED** [review]
Hey @matteo, I found couple of files which headers weer not changed (see comments bellow).

**Comment by MMaiero** [comment]
Correct. Added a commit for that. Thanks!

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**